src/widgets/InfoBadge.js

```diff
@@ -54,7 +58,7 @@ const InfoBadgeComponent = ({
           <Badge ref={ref} value={unfinalisedCountText} onPress={show} />
           <Popover
             isVisible={visible}
-            fromView={ref.current}
+            from={ref}
```

Comment:
Actually this was from a couple prs ago when I pumped `popover-view` - they changed the API